group_vars/all/netbox_clusters.yaml

```diff
@@ -0,0 +1,5 @@
+netbox_clusters:
+  - name: ovirt_cluster
+    type: virtualization
+  - name: neu-3-30_cluster
+    type: virtualization
```

Comment:
open to suggestions about these names. 
neu-3-30 is a single host running VMs and not exactly a cluster.

Reply:
What does defining it as a "cluster" get us?

Reply:
A VM needs to a part of some cluster in netbox. So we need to define a cluster, it's just a way netbox likes to group virtual machines.

Reply:
Got it.

I guess you could call it the "kaizen-bmi" cluster instead of "neu-3-30-cluster", but either seems fine.